### PR TITLE
1.5 cleanup: style fixes

### PR DESCRIPTION
**Pull request by scottdixon**

### CODE REVIEW

**Comment by scottdixon** [comment]
@moizjv @imurchie

File: package.json [hunk 1]
```diff
@@ -29,17 +29,17 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.7",
-    "appium-adb": "2.0.0-beta14",
+    "appium-adb": "2.0.0-beta15",
```
Comment (imurchie) [inline package.json]: We ought to be able to use `beta` and `latest` for our packages, while they are up in the air. The packages need to be published with the tags, though (at least, the beta ones do).

File: package.json [hunk 2]
```diff
@@ -29,17 +29,17 @@
...
     "appium-base-driver": "^0.4.1",
     "appium-chromedriver": "^2.3.0",
-    "appium-express": "^0.0.5",
+    "appium-express": "^0.0.6",
     "appium-logger": "^1.1.7",
     "appium-support": "2.0.0-beta21",
     "appium-unlock": "0.0.1",
     "asyncbox": "^2.0.4",
-    "babel-runtime": "=5.5.5",
+    "babel-runtime": "5.8.20",
```
Comment (imurchie) [inline package.json]: Is this the one we want to use? If so, we also need `=` here or we sometimes run into problems with babel

File: lib/android-helpers.js [hunk 3]
```diff
@@ -75,7 +75,7 @@ helpers.ensureDeviceLocale = async function (adb, language, locale) {
 };
 
 helpers.getActiveDevice = async function (adb, udid) {
-  logger.info('Retrieving device list');
+  logger.info("Retrieving device list");
```
Comment (imurchie) [inline lib/android-helpers.js]: Why the single quote to double?

Comment (scottdixon) [inline lib/android-helpers.js]: The code was a mixture of both, predominately doubles. I can make all singles?

Comment (imurchie) [inline lib/android-helpers.js]: Sure. But going forward this is pretty low priority. :)

File: package.json (hunk 1 above)
Comment (sebv) [inline package.json]: Yeah anything:
-beta -> change it to: beta
0.x.x -> change it  to latest or upgrade the deps to 1.0.0 (SemVer is not enable for 0.x.x packages.)

**Comment by scottdixon** [comment]
Reverted package.json changes until packages are tagged correctly. Only changes in this PR are style fixes.

**Pull request closed without merging**